lib/sbconstants/cli.rb

```diff
@@ -94,7 +94,7 @@ def write
       if options.use_swift
           swift_out = File.open("#{options.output_path}.swift", 'w') unless dry_run
           SwiftConstantWriter.new(self, swift_out, options.templates_dir).write
-          swift_out.close
+          swift_out.close unless dry_run
```

Comment:
I think you can do away with multiple conditionals here

    unless dry_run
      File.open("#{options.output_path}.swift", 'w') do |file|
        SwiftConstantWriter.new(self, file, options.templates_dir).write
      end
    end